puppet/modules/kubernetes/templates/kube-apiserver.service.erb

```diff
@@ -95,6 +95,7 @@ ExecStart=<%= scope['kubernetes::_dest_dir'] %>/apiserver \
 <% if @post_1_12 -%>
  "--service-account-signing-key-file=<%= scope['kubernetes::_service_account_key_file'] %>" \
  "--service-account-issuer=<%= @service_account_issuer %>" \
+  --service-account-api-audiences=<%= @_service_account_api_audiences.join(',') %> \
```

Comment:
I think it's was changed to `--api-audiences` in `1.13` we should use that going forward

Reply:
I think it works but it will shows a deprecation warning

Reply:
Sounds good to me :smile: